r/tools/nixlibs.R

```diff
@@ -320,10 +313,9 @@ get_macos_openssl_dir <- function() {
 # (built with newer devtoolset but older glibc (2.17) for broader compatibility, like manylinux2014)
 determine_binary_from_stderr <- function(errs) {
   if (is.null(attr(errs, "status"))) {
-    # There was no error in compiling: so we found libcurl and OpenSSL >= 1.1,
-    # openssl is < 3.0
-    lg("Found libcurl and OpenSSL >= 1.1")
-    return("openssl-1.1")
+    # There was no error in compiling: so we found libcurl and OpenSSL >= 3.0,
```

Comment:
See comment above, if you don't remove that second check, the script will error on openssl >= 3.0 so this won't be true.